website/pages/blog/graphql-yoga-v3.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Announding GraphQL Yoga v3
+tags: [graphql, nodejs, typescript, yoga]
+authors: [saihaj, arda, enisdenjo, laurin]
+date: 2022-11-15
+description: The new version of GraphQL Yoga is out! Learn what has improved and what is new!
+image: /x
+thumbnail: /x
+---
+
+After months of experimentation and learning from production use cases we are back again with a ground breaking release for GraphQL Yoga.
+
+## The Yoga v3 Experience
+
+The goal with Yoga is to empower developers, improve the developer facing API and make caching, security and real-time easy and run anywhere.
+Let's highlight some of the groundbreaking new features and improvements.
+
+### One Package to rule them all
+
+From GraphQL Yoga v2 we figure that using the `fetch` API as standard is the way to go.
+Now GraphQL Yoga has a single NPM package for all environments `graphql-yoga` instead of `@graphql yoga/common` and `@graphql-yoga/node`.
+We [created a library that lets you create cross-platform HTTP servers](https://www.the-guild.dev/blog/fetch-for-servers) which GraphQL Yoga v3 uses under the hood.
+
+[Learn more](https://the-guild.dev/graphql/yoga-server/docs)
+
+### The Makeover
+
+GraphQL Yoga v3 comes with the brand new GraphiQL 2.0
+
+![GraphiQL](https://raw.githubusercontent.com/graphql/graphiql/main/packages/graphiql/resources/graphiql.png)
+
+[Learn more](https://the-guild.dev/graphql/yoga-server/docs/features/graphiql)
+
+### 100% compliant
+
+As of today, GraphQL Yoga is only GraphQL servers in the ecosystem that fully complies with GraphQL over HTTP specification.
```

Comment:
`... is the only GraphQL server in the...`

Reply:
Link in `GraphQL over HTTP specification`

Reply:
Is the only **JavaScript** GraphQL server. [HotChocolate is fully compliant too.](https://github.com/graphql/graphql-http/blob/master/implementations/hotchocolate/README.md)

Reply:
```suggestion
As of today, GraphQL Yoga is the only GraphQL server in the JavaScript ecosystem that fully complies with GraphQL over HTTP specification.
```

Reply:
Do you guys know that god invented the suggestion feature?